Training_node/first_project/app/controllers/users.js

```diff
@@ -103,3 +103,31 @@ exports.create = (req, res, next) => {
     next(errors.defaultError(validation.reason));
   }
 };
+
+exports.getAll = (req, res, next) => {
+  if (req.query.limit && req.query.page) {
+    const limitInt = parseInt(req.query.limit);
+    const pageInt = parseInt(req.query.page);
+    if (pageInt === 1) {
```

Comment:
im asking if we need to check if the page is one and then do something, or else do other thing. i would prefer not to ask this in the controller